lib/aerospike/node.rb

```diff
@@ -69,6 +69,17 @@ def has_rack(ns, rack_id)
       racks[ns] == rack_id
     end
 
+    def connection_pool_init(min_connection_size)
```

Comment:
This is really not an init function, but more like a fill function. Change the name to something like `fill_connection_pool_up_to` or a better name. This would allow users to call it later whenever they need it, without worrying about it actually creating a new pool.